atlas-slurm

```diff
@@ -98,6 +98,10 @@ while getopts ":c:w:m:p:j:g:l:o:e:n:f:q:s:r:v:u:" o; do
     esac
 done
 
+if [[ ! -n "${workingDir}" ]]; then
+    workingDir="$scriptDir"
+fi
```

Comment:
Perhaps an additional check that the `$workingDir`, if provided, exists. If not, create it.
```suggestion
elif [[ ! -d "${workingDir}" ]]; then
    mkdir -p $workingDir
fi
```